tests/cypress/e2e/unit_tests/rbac_fleet.spec.ts

```diff
@@ -16,87 +16,66 @@ import 'cypress/support/commands';
-            { resource: "gitrepos", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
-            { resource: "bundles", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
-          ]
-        });
-
-        // Assign role to the created user
-        cy.assignRoleToUser(baseUser, customRoleName);
-
-        // Logout as admin and login as other user
-        cypressLib.logout();
-        cy.login(baseUser, uiPassword);
-
-        // Ensuring the user IS able to "go to Continuous Delivery", "list" and "delete" workspaces.
-        cy.accesMenuSelection('Continuous Delivery', 'Git Repos');
-        cy.wait(500);
-        cy.accesMenuSelection('Continuous Delivery', 'Advanced', 'Workspaces');
-        cy.accesMenuSelection('Continuous Delivery', 'Advanced', 'Bundles');
-
-        // Ensuring user cannot access Clusters nor Clusters Groups
-        cy.accesMenuSelection('Continuous Delivery');
-        cy.contains('Clusters').should('not.exist');
-        cy.contains('Clusters Groups').should('not.exist');
-
-        // FLEET-42 test cases extra check.
-        if (qase_id === 42) {
-          cy.accesMenuSelection('Continuous Delivery', 'Advanced');
-          cy.contains('Cluster Registration Tokens').should('not.exist');
-          cy.contains('GitRepoRestrictions').should('not.exist');
-          cy.contains('BundleNamespaceMappings').should('not.exist');
-        }
-        // Logout other user and login as admin user to perform user and role cleanup
-        cypressLib.logout();
-        cy.login();
-        cy.deleteUser(baseUser);
-        cy.deleteRole(customRoleName, roleTypeTemplate.toUpperCase());
-      })
-    )
-    }
+describe('Test Fleet access with RBAC with custom roles using all verbs for User-Base and Standard User.', { tags: '@rbac' }, () => {
+
+  qase(5,
+    it('Test "User-Base" role user with custom role to "fleetworkspaces", "gitrepos" and "bundles" and  ALL verbs access CAN access "Workspaces", "Bundles" and "Git Repos" but NOT "Clusters" NOR "Clusters Groups"', { tags: '@fleet-5' }, () => {
+
+      const customRoleName = "fleetworkspaces-bundles-gitrepos-all-verbs-role"
+      // Create 'base-user' User using "User-Base"
+      cypressLib.burgerMenuToggle();
+      cypressLib.createUser(baseUser, uiPassword, "User-Base", true);
+
+      cy.createRoleTemplate({
+        roleType: roleTypeTemplate,
+        roleName: customRoleName,
+        rules: [
+          { resource: "fleetworkspaces", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+          { resource: "gitrepos", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+          { resource: "bundles", verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]},
+        ]
+      });
+
+      // Assign role to the created user
+      cy.assignRoleToUser(baseUser, customRoleName);
```

Comment:
You have created 2 describe with the same boiler plate.
This is not what I suggested.
I suggested 1 `Describe` with 1 boilerplate and 2 `it` tests reusing same boilerplate, that way code re-utilization is in and is clear to read.

Reply:
I thought in that way, but later I separated tests with 2 describes based on User Base and Standard User.